src/transformStateWithClones.js

```diff
@@ -7,7 +7,28 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateHistory = [];
+  let currentState = Object.assign({}, state);
+
+  for (let i = 0; i < actions.length; i++) {
+    const action = actions[i];
```

Comment:
you can just use `for of`
```suggestion
  for (const action of actions) {
```